assets/src/scss/components/_burger-menu.scss

```diff
@@ -0,0 +1,267 @@
+.burger-menu {
```

Comment:
Can you remove all CSS here that is the same as our base CSS?

Reply:
The situation here is because they're using custom colors, like [this](https://github.com/greenpeace/planet4-child-theme-japan/pull/71/files/791133a6ead85975a9875f6b3178bdd10af3d425#diff-76fb9196f63ac67df6bd6d6632352d597817f840cdf54f599aee9d5cf475fa64R106). 

Also there are some other colors that aren't added on their [variables css file](https://github.com/greenpeace/planet4-child-theme-japan/blob/develop/assets/src/scss/base/variables.scss). But I'm thinking we can add `p4 colores` on their theme and avoid overriding the burger menu css file. What do you think @Inwerpsel ?